### ISSUE TITLE
More filters

### ISSUE THREAD

**Issue by dracoyinyang** [Elijas/auto-youtube-subscription-playlist-2#136]
is there any way we can add a filter to not add member only content, and a filter to not add live streams.

**Comment by SabreRunner** [comment]
Yes. Not add livestreams or at least recognize duplicates when it saved a livestream and then finds the ended video.
